src/api.js

```diff
@@ -1479,7 +1479,7 @@ api.declare({
     this.workerInfo.seen(task.provisionerId),
   ]);
 
-  await this.workerInfo.taskSeen(task.provisionerId, task.workerType, workerGroup, workerId, result);
+  await this.workerInfo.taskSeen(task.provisionerId, task.workerType, workerGroup, workerId, [result]);
```

Comment:
You need to read the docs for `WorkClaimer.claimTask`, see:
https://github.com/taskcluster/taskcluster-queue/blob/master/src/workclaimer.js#L203-L208

It might not return a task claim at all...